src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -118,37 +127,40 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-
-.wrapper {
-	width: 100%;
-	padding: 0 0 0 0;
-	&:focus {
-		background-color: rgba(47, 47, 47, 0.068);
-	}
-}
+@import '../../../../assets/variables';
 
 .message {
-	display: flex;
-	max-width: 600px;
-	padding: 8px 0 8px 0;
-	margin: auto;
-	&-avatar {
-		width: 52px;
-		min-width: 52px;
-		padding: 4px 8px 0 8px;
+	padding: 4px 0 4px 0;
+	flex-direction: column;
+	&__author {
+		color: var(--color-text-maxcontrast);
 	}
-	&-right {
+	&__main {
 		display: flex;
-		min-width: 110px;
-		color: #989898;
-		padding: 0 8px 0 8px;
-	}
-
-	.actions {
-		position: absolute;
-		margin: -14px 0 0 50px;
-		padding:2px;
+		justify-content: space-between;
+		min-width: 100%;
+		&__text {
+			flex: 1 1 400px;
+			color: var(--color-text-light);
+			&--quote {
+			border-left: 4px solid var(--color-primary);
+			padding: 4px 0 0 8px;
+			}
+		}
+		&__right {
+			justify-self: flex-start;
+			position: relative;
+			flex: 0 0 110px;
+			display: flex;
+			color: var(--color-text-maxcontrast);
+			font-size: 13px;
+			padding: 0 8px 0 8px;
+			&__actions {
+				position: absolute;
+				top: -50%;
```

Comment:
My mistake, you need to switch off from 12px.
Time is 20px high, 44-20 = 24, /2 = 12px :)
```suggestion
				top: -12px;
```

Reply:
Maybe you could set a specific height for the time, so it's understandable on the code?